File: LeetCode/1108-Defanging an IP Address/solution.cpp
class Solution {
public:
    string defangIPaddr(string address) {
        string defangedIPAddress = "";
            
        for (int i = 0; i < address.size(); i++) {
            if (address[i] == '.')
                defangedIPAddress += "[.]";
            else
                defangedIPAddress += address[i];
        }
        return defangedIPAddress;
    }
};

File: LeetCode/1108-Defanging an IP Address/solution.rs
impl Solution {
    pub fn defang_i_paddr(address: String) -> String {
        let mut defangedString = String::from("");

        for i in address.chars() {
            if i == '.' {
                defangedString.push_str("[.]");
            } else {
                defangedString.push(i);
            }
        }
        return defangedString;
    }
}